feat(my-account): added troubleshooting page MTA-2301 (#927)
scaleway/docs-content: console/my-account/troubleshooting/cant-log-into-my-account.mdx

---
meta:
  title: I cannot log into my account
  description: This page helps you log into your account
content:
  h1: I cannot log into my account
  paragraph: This page helps you log into your account
dates:
  validation: 2022-10-19
  posted: 2022-10-19
---

> Sometimes you may have problems when connecting to your Scaleway account. If you have encountered any of the following possible cases, check out our solutions.

Reviewer: ```suggestion
Sometimes you may have problems when connecting to your Scaleway account. If you have encountered any of the following possible cases, check out our solutions.
```

Can we change this "Sometimes you may have problems" sounds a bit like there could be issues :)

## Unrecognized IP address error 
You may try to log into your Scaleway account and see a message like the following: 

<Lightbox src="scaleway-unrecognized-ip.webp" alt="" />

> This means you tried to log into your account from an unknown IP address. As a security measure, we require you to use a magic link to complete authentification. 

Reviewer 2: Since this doc is aimed at people who might not be that tech-savvy, I wonder if we should add something like  "This suggests you might be trying to log in from a new device or location. As a security measure..."

 
1. Check your inbox for the latest email sent by the Scaleway team.

> 2. Click on the link provided in the email. It should be a button titled **"Authenticate Myself"**.

Reviewer 3: ```suggestion
2. Click the link provided in the email. It should be a button titled **"Authenticate Myself"**.
```

> 3. You're safely logged into your Scaleway account. 

Reviewer 3: ```suggestion
3. You are safely logged into your Scaleway account. 
```

> 3. You're safely logged into your Scaleway account. 

Reviewer 2: Every step should start with an action verb that represents something the user has to do. I'd suggest after step 2 to just do a linebreak and text: You are redirected to the Scaleway console, and safely logged into your Scaleway account.

## I have forgotten my password

> If you forgot or lost your password, you need to reset it. There are several methods to retrieve it :

Reviewer: ```suggestion
If you forgot or lost your password, you need to reset it. There are several methods to retrieve it:
```

>   - [Passwordless authentification using a Magic Link](https://www.scaleway.com/en/docs/console/my-account/how-to/recover-a-lost-password/#passwordless-authentication-using-a-magic-link)

Reviewer 3: ```suggestion
  - [Passwordless authentication using a Magic Link](https://www.scaleway.com/en/docs/console/my-account/how-to/recover-a-lost-password/#passwordless-authentication-using-a-magic-link)
```

>   - [Passwordless authentification using a Magic Link](https://www.scaleway.com/en/docs/console/my-account/how-to/recover-a-lost-password/#passwordless-authentication-using-a-magic-link)

Reviewer 2: ```suggestion
  - [Passwordless authentification using a Magic Link](/console/my-account/how-to/recover-a-lost-password/#passwordless-authentication-using-a-magic-link)
```

>   - [Password reset using email validation](https://www.scaleway.com/en/docs/console/my-account/how-to/recover-a-lost-password/#password-reset-using-email-validation)

Reviewer 2: ```suggestion
  - [Password reset using email validation](/console/my-account/how-to/recover-a-lost-password/#password-reset-using-email-validation)
```

>   - [Password recovery in case of lost email account](https://www.scaleway.com/en/docs/console/my-account/how-to/recover-a-lost-password/#password-recovery-in-case-of-lost-email-account)

Reviewer 2: For links that are internal to the doc site, you don't need to use the whole address, just the part after https://www.scaleway.com/en/docs. Make sure they start with a slash.

>   - [Password recovery in case of lost email account](https://www.scaleway.com/en/docs/console/my-account/how-to/recover-a-lost-password/#password-recovery-in-case-of-lost-email-account)

Reviewer 2: ```suggestion
  - [Password recovery in case of lost email account](/console/my-account/how-to/recover-a-lost-password/#password-recovery-in-case-of-lost-email-account)
```

## I have lost my 2FA device

> Refer to the [How to use two-factor authentication (2FA)](https://www.scaleway.com/en/docs/console/my-account/how-to/use-2fa/) page to find the instructions to retrieve access to your 2FA device. 

Reviewer 2: ```suggestion
Refer to the [How to use two-factor authentication (2FA)](/console/my-account/how-to/use-2fa/) page to find the instructions to retrieve access to your 2FA device. 
```

> Remember that in case of losing both your 2FA device and back up codes, you will need to gather all the following information :

Reviewer: ```suggestion
Remember that in case of losing both your 2FA device and back up codes, you will need to gather all the following information:
```

> Remember that in case of losing both your 2FA device and back up codes, you will need to gather all the following information :

Reviewer 3: ```suggestion
Remember that in case you lost both your 2FA device and back up codes, you will need to gather all the following information:
```

  - Email address
  - Postal address
  - Two different documents proving your identity (passport or local ID card for individual accounts + certificate of incorporation for companies)
  - A contact phone number 
  - A copy of your last Scaleway invoice, or details about the amount charged if you do not have a copy 
  - A card or bank statement showing your Scaleway invoice debit 

Once you have collected all this information, send it to `document@scaleway.com`.

> 

Reviewer: Pourquoi t'as mis autant des new lines a la fin ? Une ligne est suffisant :)

Author: Good catch - j'ai pas fait attention 😄
